build/template-build-and-run-all-tests.yaml

```diff
@@ -107,21 +107,20 @@ jobs: #Build and stage projects
     parameters:
       DataFileDirectory: ${{ parameters.DataFileDirectory }}
 
-# Disabled because mono is unable to restore/build .NET 6 target in MSAL project
 # APP CENTER IOS MSAL TESTS PHASE
-#- job: 'RunAppCenterIosMsalTests'
-#  dependsOn:
-#  - 'BuildAndStageProjects'
-#  - 'BuildiOSAppCenterApp'
-#  pool:
-#    vmImage: 'windows-2022'
-#  variables:
-#    runCodesignValidationInjection: false
-
-#  steps:
-#  - template: template-ios-appcenter-tests.yaml
-#    parameters:
-#      DataFileDirectory: ${{ parameters.DataFileDirectory }}
+# - job: 'RunAppCenterIosMsalTests'
```

Comment:
so iOS tests are still not working?

Reply:
iOS started failing again. I tried several times with no luck. If your branch has fix, that can be merged.